### PR TITLE
tests for upload that would exceed the quota

### PR DESCRIPTION
**Pull request by individual-it**
## Description
Upload a file that would exceed the set quota

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/helpers/httpHelper.js [hunk 1]
```diff
@@ -18,3 +18,11 @@ exports.createAuthHeader = function (userId) {
       Buffer.from(userId + ':' + password).toString('base64')
   }
 }
+
```
Comment (paurakhsharma) [inline tests/acceptance/helpers/httpHelper.js]: Please add docstring.

Comment (individual-it) [reply tests/acceptance/helpers/httpHelper.js]: @paurakhsharma done

**Comment by paurakhsharma** [comment]
Oops, you committed test/report

**Comment by individual-it** [comment]
@paurakhsharma :man_facepalming: 
fixed it

**Pull request merged**